packages/contracts/contracts/foundry/SphinxConstants.sol

```diff
@@ -10,3 +10,43 @@ contract SphinxConstants {
   address public constant safeSingletonAddress = 0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552;
   address public constant sphinxModuleImplAddress = 0x8f4E4d51B8050B0ff713eff1F88f3dD8b5e8a530;
 }
+
+enum Network {
```

Comment:
This is now automatically generated from the `SPHINX_NETWORKS` object in the `write-constants.ts` script.